unleash-yggdrasil/src/strategy_grammar.pest

```diff
@@ -3,9 +3,7 @@ num = @{ int ~ ("." ~ ASCII_DIGIT*)? ~ (^"e" ~ int)? }
 
 percentage = { ("100" | (ASCII_DIGIT ~ ASCII_DIGIT?)) ~ "%" }
 
-string = @{ "\"" ~ ( unicode_characters | punctuation_characters )* ~ "\"" }
-    unicode_characters = { LETTER | MARK | NUMBER | SYMBOL | SEPARATOR | DASH_PUNCTUATION }
-    punctuation_characters = { "." | "&" | "/" | "_" | "@" }
+string = @{ "\"" ~ ( !("\\\"" | "\"") ~ ANY | "\\" ~ "\"")* ~ "\"" }
```

Comment:
Reads loosely as match anything that starts with a quote but allow escaping quotes through \" characters. This should tolerate more or less anything unicodey and correctly escape the other grammar fragments